TF-10033 Update the CLI-driven Run Workflow documentation (#487)
hashicorp/terraform-docs-common: website/docs/cloud-docs/run/cli.mdx

- New workspaces are not automatically connected to a VCS repository and do not have a working directory specified.
- A new workspace's Terraform version defaults to the most recent release of Terraform at the time the workspace was created.

### Implicit Project Creation

> If you configure the [`workspaces` block](/terraform/cli/cloud/settings#workspaces) to use a [project](/terraform/cli/cloud/settings#project) that does not yet exist in your organization, Terraform Cloud will create a new project with that name when you run `terraform init` and notify you in the command output.

Reviewer: ```suggestion
If you configure the [`workspaces` block](/terraform/cli/cloud/settings#workspaces) to use a [project](/terraform/cli/cloud/settings#project) that does not yet exist in your organization, Terraform Cloud will attempt to create a new project with that name when you run `terraform init` and notify you in the command output.
```

> If you configure the [`workspaces` block](/terraform/cli/cloud/settings#workspaces) to use a [project](/terraform/cli/cloud/settings#project) that doesn't yet exist in your organization, Terraform Cloud will create a new project with that name when you run `terraform init`. The output of `terraform init` will inform you when this happens.
> 
> If both the project argument and [`TF_CLOUD_PROJECT`](/terraform/cli/cloud/settings#tf_cloud_project) environment variable are specified, the `cloud` block configuration takes precedence.
> 

Reviewer 2: ```suggestion
If you configure the [`workspaces` block](/terraform/cli/cloud/settings#workspaces) to use a [project](/terraform/cli/cloud/settings#project) that does not yet exist in your organization, Terraform Cloud will create a new project with that name when you run `terraform init` and notify you in the command output.

If you specify both the `project` argument and [`TF_CLOUD_PROJECT`](/terraform/cli/cloud/settings#tf_cloud_project) environment variable, the `project` argument takes precedence.

```
